woocommerce/sensei-content-drip
# Add partial French translation

### Changes proposed in this Pull Request

* Adds partial French translation submitted by a user.

### Testing instructions

* Change the site language to French.
* Edit a lesson and view Content Drip settings.
* Ensure "When should this lesson become available?" text is translated.
